script.js

```diff
@@ -0,0 +1,47 @@
+const mybutton = document.getElementById("scroll-on-top");
+    window.onscroll = () => {
+        if (document.body.scrollTop > 10 ||document.documentElement.scrollTop > 10) 
```

Comment:
cài mấy plugin format code vào 
```suggestion
        if (document.body.scrollTop > 10 || document.documentElement.scrollTop > 10) 
```